src/workflows/rna/rna_singlesample/integration_test.sh

```diff
@@ -11,6 +11,7 @@ cd "$REPO_ROOT"
 export NXF_VER=21.10.6
 
 viash ns build -q rna_singlesample
+# viash ns build -q 'filter|publish|qc|metadata' --parallel --setup cb
```

Comment:
Should this be removed?

Reply:
can be. The second command is a little more precise when building containers.